### DISCUSSION TITLE
Question - does DDEV need to start the web container to perform a snapshot?

### DISCUSSION THREAD

**Discussion by lozcalver** [ddev/ddev#5559] [General]
Sorry if this has already been raised, I searched for it but couldn’t find anything.

One of the machines I use for development has a slower hard disk, so Mutagen sync can take quite some time. When doing snapshots, e.g. `ddev snapshot --all` to snapshot every project I’m working on for backups, DDEV starts + stops both the `db` and `web` containers, with the latter taking a while due to Mutagen.

Is it actually necessary to start the `web` container, or would it be possible to just start the `db` container?

**Comment by rfay** [comment]
I don't think the logic is perfect there; theoretically it should just be able to restart the db container, but I don't think it actually works that way.

However, if mutagen is taking a long time to sync, there's something wrong with your setup, I'd be happy to take a look at that with you. Essentially you need to use `ddev mutagen monitor` (in another window) to find out what it's having trouble with, and those directories may need to be added to `upload_dirs`.
